packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -239,11 +239,7 @@ mixin SingleTickerProviderStateMixin<T extends StatefulWidget> on State<T> imple
     _updateTicker();
   }
 
-  void _updateTicker() {
-    if (_ticker != null) {
-      _ticker!.muted = !_tickerModeNotifier!.value;
-    }
-  }
+  void _updateTicker() => _ticker?.muted = !_tickerModeNotifier!.value;
```

Comment:
Personally I prefer reserving `=>` only for getters. So for this one, 
```suggestion
  void _updateTicker() {
    _ticker?.muted = !_tickerModeNotifier!.value;
  }
```

Reply:
One of the goals of the style guide is "to avoid arguments when there are disagreements over subjective matters."

With regards to the `=>` syntax, the only condition listed there is that [everything should fit on one line](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#consider-using--for-short-functions-and-methods).

Reply:
Well, the style guide does list both styles as GOOD.

I'm not strongly against the `=>` in non-getters, and I wouldn't really point out if a fresh PR is written this way, but I'm not a fan of changing the code from one way to another when both are listed as acceptable. But if you feel strong about it, we can make this change this time.